docs/process/release.md

```diff
@@ -76,9 +76,18 @@ or a separate PR after FW releases but before CLI releases.
 
 **For a major release:**
 
-When a release is transitioning from a prerelease to a stable release, the semver ranges for Angular dependencies within the packages' `package.json` files will need to be updated to remove the prerelease version segment.
-For example, `"@angular/compiler-cli": "^13.0.0 || ^13.0.0-next"` in a prerelease should become `"@angular/compiler-cli": "^13.0.0"` in the stable release.
-The current packages that require adjustment are:
+**As part of the release PR**, the
+[`latest-versions.ts`](https://github.com/angular/angular-cli/blob/main/packages/schematics/angular/utility/latest-versions.ts#L18)
+file needs the Angular semver to be updated from `~13.0.0-rc` to just `~13.0.0`. This is the version
+generated in `ng new` schematics and needs to be updated to avoid having users generate projects with
+`-rc` in the dependencies.
+
```

Comment:
Can you also please mention to check that `ng-packagr` is also update to a stable version?

https://github.com/angular/angular-cli/blob/e569d3bfdf9629dfcc8ab9acaae8de83a605ef5e/packages/schematics/angular/utility/latest-versions/package.json#L15

Reply:
Done, thanks for pointing that out!